src/components/DataView/components/DataViewBody/DataViewBody.tsx

```diff
@@ -53,7 +53,12 @@ const DataViewBody = ({
                   key={`${JSON.stringify(column)}${idx}`}
                   data-testid="data-view-body-row-value"
                 >
-                  <StyledImg alt="Icoon" src={row[column]} />
+                  <StyledImg
```

Comment:
Styling van componenten van DataViewBody zit in de styled.js van DataView. Ik zou suggeren om dit daarbij te doen (en als je helemaal cool bent, dan ook gelijk het om te zetten, maar ik ga binnenkort DataView omzetten en dan kan ik het ook doen als je wil)

Reply:
These are properties that should stay there. They resize the svg file.